### DISCUSSION TITLE
How to implement offline mode payment

### DISCUSSION THREAD

**Discussion by khiemvu** [flutter-stripe/flutter_stripe#1809] [Q&A]
**Is your feature request related to a problem? (please describe)**
A clear and concise description of what the problem is. Ex. I would like to see [...]

**Describe the solution you'd like**

**Describe alternatives you've considered**
A clear description of  alternative solutions or features you've considered.

**Additional context**

**Comment by htduongdl96** [comment]
I also have this issue, in the documents it said that Stripe SDK already support this feature.

**Comment by remonh87** [comment]
the offline payments are related to the Stripe terminal SDK. For normal app payments you always need to be online this is how Stripe works
